Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -266,7 +266,11 @@ private extension LoginListViewController {
 
     func loadLogins(_ query: String? = nil) {
         loadingView.isHidden = false
-        viewModel.loadLogins(query, loginDataSource: self.loginDataSource)
+        viewModel.loadLogins(query, loginDataSource: self.loginDataSource)?.upon { _ in
+            DispatchQueue.main.async {
+                self.tableView.reloadData()
```

Comment:
I think this is the weakest part of my code, but it works. Any suggestions to improve the way I'm going about this (to reload only the affected cells and not the entire table) would be appreciated.